integrations/acquisition/covidcast/test_delete_batch.py

```diff
@@ -25,37 +25,22 @@ class DeleteBatch(unittest.TestCase):
     def setUp(self):
```

Comment:
minor: wondering if it would be worth it to take this boiler plate `setUp`, `tearDown` common testing stuff and refactor it into a common file, and have all the integration test classes inherit.

Reply:
#897